cypress/integration/sizesAutoSpec.js

```diff
@@ -0,0 +1,76 @@
+describe('On a pages first render', () => {
+  before(() => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+  });
+  beforeEach(() => {
+    cy.fixture('config.js').as('config');
+  });
+
+  it('Sizes attribute is correctly set', () => {
+    cy.get('.sizes-test', { timeout: 10000 }).each(($el) => {
+      const expectedSize = Math.ceil($el.width());
+      const imgSize = Number($el.attr('sizes').split('px')[0]);
+      assert.isAtMost(imgSize, 500);
+      assert.isAtLeast(imgSize, expectedSize);
+    });
+  });
+});
+
+describe('When a page gets resized', () => {
+  before(() => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+  });
+
+  beforeEach(() => {
+    cy.fixture('config.js').as('config');
+    cy.viewport(500, 500);
+  });
+
+  it('Updates the sizes attribute on resize', () => {
```

Comment:
I might be missing something, but is there supposed to be a difference between this test and the on [L9](https://github.com/imgix/imgix.js/pull/284/files#diff-fa15396007957bc953c6939288a000756461b192cf5a7ac2a17bbec875ef9196R9)?

Reply:
Just comes down to [L26](https://github.com/imgix/imgix.js/pull/284/files#diff-fa15396007957bc953c6939288a000756461b192cf5a7ac2a17bbec875ef9196R26). There's a viewport resize happening before each test block in this describe loop, so, in theory, a resize event should have triggered and the `autoSize` module should have updated `sizes` accordingly.

L9 only tests for first render attribute updates, where `sizes` is set to the current viewport size _before_ any resize events have actually fired.

Reply:
As far as `dist/`, there was an issue with the way I had previously setup cypress fixtures. Tests would always fail CI since `dist/` was not update before test suites ran.

This was fixed in #293, where `dist/` checkins are rolled back and we also introduce script updates to ensure `npm run build` is run before any test suite runs.

Reply:
> Just comes down to L26

Ah, totally grazed over that. And thanks for explaining the `dist/` thing, especially before I go and leave the same comment on six other PRs 😂